openassessment/xblock/static/js/src/studio/oa_edit_listeners.js

```diff
@@ -95,7 +95,7 @@ OpenAssessment.StudentTrainingListener.prototype = {
         if (criterionAdded) {
             this.displayAlertMsg(
                 gettext("Criterion Added"),
-                gettext("You've added a criterion. You'll need to select an option for the criterion in the Student Training step. To do this, click the Settings tab.")
+                gettext("You've added a criterion. You'll need to select an option for the criterion in the Learner Training step. To do this, click the Settings tab.")
```

Comment:
"You've added..." -> "You have added..."
"You'll need to select..." -> "You will need to select..."